src/components/CardFilm.js

```diff
@@ -209,4 +204,19 @@ Item.propTypes = {
   }),
 };
 
-export default connect()(Item);
+const mapStateToProps = (state) => ({
+  sortType: state.movieReducer.sort,
+  movies: state.movieReducer.movies,
+});
+
+const mapDispatchToProps = (dispatch) => {
```

Comment:
Just a notice: `mapDispatchToProps` could be just an object:
```
const mapDispatchToProps = {
  setEditFilm,
  fetchMoviesByGenre,
  fetchMovieId,
}
```
connect will automatically wrap this functions with `dispatch` and pass into props with the same keys.
If you need to change keys (for any reason), you could do it like:
```
const mapDispatchToProps = {
  instEditFilm: setEditFilm,
  getMoviesByGenre: fetchMoviesByGenre,
  getMovieById: fetchMovieId,
}

Reply:
fix